src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const cleanedSourceString = sourceString.replace(/\s*;\s*/g, ';').trim();
+  const strArray = cleanedSourceString.split(';');
+  const result = {};
```

Comment:
The variable name `result` is too generic. A more descriptive name such as `stylesObject` would improve readability.